static/app/views/settings/components/dataScrubbing/modals/form/index.tsx

```diff
@@ -133,10 +131,10 @@ class Form extends Component<Props<Values, KeysOfUnion<Values>>, State> {
           </Field>
           {values.type === RuleType.PATTERN && (
             <Field
-              data-test-id="regex-field"
               label={t('Regex matches')}
               help={t('Custom regular expression (see documentation)')}
               inline={false}
+              id={t('Regular expression')}
```

Comment:
wait this should be an id string hahah, not a translation

Reply:
damn!

Reply:
like `id="regexMatches"` or whatever

Reply:
thanks. Updated 😄